test/helper/sample-schema/books/books.go

```diff
@@ -93,7 +93,7 @@ func ClassMixedContextionaryVectorizer() *models.Class {
 		"contextionary_all": {
 			Vectorizer: map[string]interface{}{
 				"text2vec-contextionary": map[string]interface{}{
-					"vectorizeClassName": false,
+					"vectorizeClassName": true,
```

Comment:
why do we need to vectorize class name?

Reply:
I wanted the named vector to be exactly the same as the legacy vector so I could check whether they match.

And the legacy vector is configured to vectorize the class name: 
https://github.com/weaviate/weaviate/blob/bd854784460c34e71ee0df0a43ede0bee50e0400/test/helper/sample-schema/books/books.go#L166-L171